src/components/dev-hub/stories/theme.stories.mdx

```diff
@@ -42,8 +43,9 @@ import { colorMap, gradientMap, size, fontSize } from '../theme';
     <Story name="gradients">
         {() => {
             const Wrapper = styled('div')`
-                fontfamily: 'Fira Mono', monospace;
-                fontsize: ${fontSize.xsmall};
+                color: #fff;
```

Comment:
Same comments as above, but I'm curious why we need to define these styles in multiple places. Shouldn't the theme be global for storybook?